Web UI/CognitiveSearch.UI/wwwroot/css/site.css

```diff
@@ -375,17 +376,22 @@ pre {
 }
 
 .results-body {
-    padding-left: 0;
+    padding-left: 20px;
     padding-right: 0;
+    padding-bottom: 100%;
+    margin-bottom: -100%;
 }
 
 .results-icon {
     padding-left: 0;
     padding-right: 0;
     text-align: center;
-    width: 30px;
-    height: 40px;
-    margin-top: 10px;
+    width: 60px;
+    height: 60px;
+    margin-top: -5px;
```

Comment:
rewrite using margin shorthand property, similar to the padding property.
Ref: https://www.w3schools.com/css/css_margin.asp